packages/o365_metrics/data_stream/outlook_activity/fields/fields.yml

```
@@ -0,0 +1,35 @@
+- name: o365metrics.outlookactivity
+  type: group
+  fields:
+    - name: meeting_created
+      type: integer
+      description: |
+        The count of calendar meetings created by users during the reporting period.
+    - name: meeting_interacted
+      type: integer
+      description: |
+        The count of meetings where users interacted (e.g., accepted, declined, or modified) during the reporting period.
+    - name: read
+      type: integer
+      description: |
+        The count of email messages read by users during the reporting period.
+    - name: receive
+      type: integer
+      description: |
+        The count of email messages received by users during the reporting period.
+    - name: send
+      type: integer
+      description: |
+        The count of email messages sent by users during the reporting period.    
+    - name: report_date
+      type: date
+      description: |
+        The specific date for which the report data applies..
```

The specific date for which the report data applies..
```suggestion
        The specific date for which the report data applies.
```